go-controller/pkg/ovn/libovsdbops/switch.go

```diff
@@ -36,6 +44,22 @@ func findSwitch(nbClient libovsdbclient.Client, lswitch *nbdb.LogicalSwitch) err
 	return nil
 }
 
+// FindSwitch Looks up switches in the cache based on the lookup function
+func FindSwitch(nbClient libovsdbclient.Client, lookupFunction func(item *nbdb.LogicalSwitch) bool) ([]nbdb.LogicalSwitch, error) {
```

Comment:
In general I would prefer to have `Find<X>By<Y>`kinda methods to facilitate reuse and avoid using predicates unless a very specific need. Otherwise I would name this `FindSwitchByPredicate`

Reply:
Done Updated the find functions for both switches and ACLs to follow this naming convention